contracts/ControlledGateway.sol

```diff
@@ -0,0 +1,34 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {Pausable} from "@openzeppelin/contracts/security/Pausable.sol";
+import {AccessControl} from "@openzeppelin/contracts/access/AccessControl.sol";
+
+contract ControlledGateway is AccessControl, Pausable {
+    bytes32 public constant GOVERNANCE_CONTROLLER =
+        keccak256("GOVERNANCE_CONTROLLER");
+
+    address public token;
```

Comment:
This can be marked with `immutable` since it is only set once in the constructor so that the value set in the constructor gets inlined into the contract's runtime code which should save some gas on variable reads.

Reply:
ah this is just a placeholder variable, the next PR does address this by making the token variables immutable
https://github.com/livepeer/arbitrum-lpt-bridge/pull/14/files#diff-5530766379b25714529b779f799166a9438bc86b5b73a476a41eddb385db10aaR11-R12

can make the change here as well if you say

Reply:
👍  to change in next PR.